### ISSUE TITLE
[Bug]: Appstream Metadata Doesn't Validate

### ISSUE THREAD

**Issue by damaestro** [WiVRn/WiVRn#210]
### Describe the bug

stock icon is not valid [io.github.wivrn.wivrn]

### Steps to Reproduce

appstream-util validate-relax --nonet io.github.wivrn.wivrn.metainfo.xml
/io.github.wivrn.wivrn.metainfo.xml: FAILED:
• tag-invalid           : stock icon is not valid [io.github.wivrn.wivrn]

### WiVRn Version

v0.22

### What GPU did the bug occur on?

NVIDIA

### What HMD did the bug occur on?

HTC Vive Focus 3

**Comment by damaestro** [comment]
```
diff -up ./io.github.wivrn.wivrn.metainfo.xml.orig ./io.github.wivrn.wivrn.metainfo.xml
--- ./io.github.wivrn.wivrn.metainfo.xml.orig	2024-12-08 13:42:38.056954454 -0700
+++ ./io.github.wivrn.wivrn.metainfo.xml	2024-12-08 13:42:44.592201655 -0700
@@ -66,7 +66,7 @@
 	<url type="bugtracker">https://github.com/WiVRn/WiVRn/issues</url>
 	<url type="donation">https://opencollective.com/wivrn</url>
 
-	<icon type="stock">io.github.wivrn.wivrn</icon>
+	<icon type="local">/usr/share/icons/hicolor/scalable/apps/io.github.wivrn.wivrn.svg</icon>
 
 	<content_rating type="oars-1.1" />
```

**Comment by xytovl** [comment]
fixed by 258150967c136c57265f909d1e7491a0a366d599
